test/common/router/router_test.cc

```diff
@@ -6351,6 +6351,18 @@ TEST_F(RouterTest, PostExplicitTcpUpstream) {
   router_.onDestroy();
 }
 
+TEST_F(RouterTest, PostHttpUpstream) {
+  // Make sure POST request result in the HTTP pool.
```

Comment:
Sorry, can you add the same config above, without set_allow_post?
Basically I want this to verify your new "use connect" doesn't trigger with connect config present but allow_post false.

Reply:
done.